src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -1086,7 +1086,7 @@ private function registerTranslatorConfiguration(array $config, ContainerBuilder
             return;
         }
 
-        $loader->load('translation.xml');
+        $loader->load('translation.php');
```

Comment:
```suggestion
        $phpLoader->load('translation.php');
```

Reply:
Hii @rvanlaak, thanks for your suggestion but i think it's not necessary to change the variable name because it's in the private method `registerTranslatorConfiguration` where no other loaders are required.

At the end i think the XmlFileLoader will be removed and the `$phpLoader` variables will be renamed to `$loader` again.